orga/app/controllers/authentication/oidc/login.js

```diff
@@ -26,15 +27,35 @@ export default class OidcLoginController extends Controller {
     const identityProviderSlug = this.model.identity_provider_slug;
     const identityProvider = this.oidcIdentityProviders.findBySlug(identityProviderSlug);
 
-    const authenticationRequest = this.store.createRecord('user-oidc-authentication-request', {
-      password,
-      email,
-      authenticationKey: this.authenticationKey,
-      identityProvider: identityProvider.code,
+    const response = await fetch(`${ENV.APP.API_HOST}/api/oidc/user/check-reconciliation`, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        data: {
+          attributes: {
+            email,
+            password,
+            'authentication-key': this.authenticationKey,
+            'identity-provider': identityProvider.code,
+          },
+        },
+      }),
     });
 
-    const oidcAssociationConfirmationData = await authenticationRequest.login();
-    oidcAssociationConfirmationStorage.set({ ...oidcAssociationConfirmationData, email });
+    const json = await response.json();
```

Comment:
```suggestion: 
const reconciliationJsonData = await response.json();
```
Pour avoir un nom plus parlant quant au contenu de ce json

Reply:
ok!